src/components/ComboBox/ComboBoxItem.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import { autobind } from '@shopify/javascript-utilities/decorators';
+
+import Label from '../Label';
+import Accordion from '../Accordion';
+
+export interface ComboItemProps {
+  type? : string;
```

Comment:
@ankitsaraogi This needs to be strictly defined.  Ex: `type?: 'accordian' | undefined`

Reply:
@ankitsaraogi Is this similar to the ComboBoxProps above?

Reply:
@harsimranb : I have kept as string, so that we can add more components later on. Also on changing it I am getting ts errors.

Reply:
@harsimranb I have made the changes, errors were coming because of typecast